utils/build/docker/dotnet/Controllers/DebuggerController.cs

```diff
@@ -53,37 +53,11 @@ public IActionResult MixProbe(string arg, int intArg)
             return Content($"Mixed result {intMixLocal}");
         }
 
-        [HttpGet("log/pii/{arg}")]
+        [HttpGet("pii")]
```

Comment:
Could you document this endpoint in `docs/weblog` ?

Reply:
Oh, sorry. Added here 45c4c3fad188f5dade41726930b75754bcaff2ef